sql/003_views.sql

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Nice trick with moving Views, could you add liquibase metadata at the header? If you add `runOnChange:true` clause, Liquibase will re-run view if the file changes, does not have to be this way.

Reply:
I can't get this to work consistently, so I haven't added it.  I have added a comment as to whys the difference between this version of the view and the `views.sql` version

Reply:
I did work for me on Liquibase 4.13 (bit aged version). 
```
--liquibase formatted sql
--changeset liquibase:create -multiple-tables:1 splitStatements:true endDelimiter:; runOnChange:true
```
with the header in the file, any change to the file triggers re-run.

Reply:
Ah! I had it below the comments....